packages/rrweb/src/replay/index.ts

```diff
@@ -1184,14 +1239,19 @@ export class Replayer {
                  * removal and addition of .active class (along with void line to trigger repaint)
                  * triggers the 'click' css animation in styles/style.css
                  */
-                this.mouse.classList.remove('active');
-                void this.mouse.offsetWidth;
-                this.mouse.classList.add('active');
+                this.pointers[pointerId].pointerEl.classList.remove('active');
+                void this.pointers[pointerId].pointerEl.offsetWidth;
+                this.pointers[pointerId].pointerEl.classList.add('active');
               } else if (d.type === MouseInteractions.TouchStart) {
-                void this.mouse.offsetWidth; // needed for the position update of moveAndHover to apply without the .touch-active transition
-                this.mouse.classList.add('touch-active');
+                void this.pointers[pointerId].pointerEl.offsetWidth; // needed for the position update of moveAndHover to apply without the .touch-active transition
+                this.pointers[pointerId].pointerEl.classList.add(
+                  'touch-active',
+                );
               } else if (d.type === MouseInteractions.TouchEnd) {
-                this.mouse.classList.remove('touch-active');
+                this.pointers[pointerId].pointerEl.classList.remove(
+                  'touch-active',
+                );
+                // delete this.pointers[pointerId];
```

Comment:
adding this `delete` here sadly caused some things to break - it seemed to want to reference a deleted pointer

Reply:
😬 we should look into what's trying to access it after this event occurs